src/ZoneServer/Database/ZoneDb.cs

```diff
@@ -883,7 +883,7 @@ private void SaveChatMacros(Account account)
 		private void LoadRevealedMaps(Account account)
 		{
 			using (var conn = this.GetConnection())
-			using (var mc = new MySqlCommand("SELECT * FROM `revealedmaps` WHERE `accountId` = @accountId", conn))
+			using (var mc = new MySqlCommand("SELECT * FROM `revealedMaps` WHERE `accountId` = @accountId", conn))
```

Comment:
Same problem as with chat macros.

Reply:
The changes to this class have been removed.